api/controller/logs.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package percona.platform.dbaas.controller.v1beta1;
+
+option go_package = "controller;controllerv1beta1";
+
+// Log contains log for certain pod's container.
+message Log {
+  // Pod name.
+  string pod = 1;
+  // Container name.
+  string container = 2;
+  // Content of container's log.
+  string log = 3;
```

Comment:
I think it makes sense to use `repeated string` there, so the caller should not know about line endings

Reply:
Done.